main.rb

```diff
@@ -1,21 +1,15 @@
 require_relative 'app'
+require_relative 'menu'
 
 app = App.new
+menu = Menu.new
 
 puts "Welcome to School Library App! \n\n"
 
 loop do
-  puts 'Choose an option:'
-  puts '1 - List all books'
-  puts '2 - List all people'
-  puts '3 - Create a person'
-  puts '4 - Create a book'
-  puts '5 - Create a rental'
-  puts '6 - List all rentals for a given person id'
-  puts '7 - Exit'
-  option = gets.chomp
+  menu.display_options
```

Comment:
- [ ] I see you probably forgot the single responsibility rule, in your methods in the app.rb file, try to make all requests for input and input validation to be placed in the main.rb since that is where input and the validation should be.
![image](https://github.com/migo1/school-library/assets/86200356/3c99b945-b804-4d40-b9da-705ad9cdef2c)

Reply:
@Olamarx  noted, working on it